### PR TITLE
new embeddings

### PR DESCRIPTION
**Pull request by mohammadsanaee**

### CODE REVIEW

File: reverse_image_search_module/search_image.py [hunk 1]
```diff
@@ -1,29 +1,48 @@
...
+    if isinstance(x, list) and len(x) > 0:
+        return x[0]['path'].split('/')[-1]
+    else:
+        return None
+
+dataset['file_name'] = dataset['images'].apply(extract_file_name)
 
 def change_format(data):
+    # image_path = data['image_urls'][0]['path']
+    # file_name = image_path.split('/')[-1]
```
Comment (aguschin) [inline reverse_image_search_module/search_image.py:45]: remove this?

File: reverse_image_search_module/search_image.py [hunk 2]
```diff
@@ -39,21 +58,47 @@ def change_format(data):
         'dimension': data['Dimensions'],
         'description': data['WikiDescription'],
         'tags': data['Tags'],
-        'image_url': data['image_urls']
+        'image_url': data['image_urls'],
+        # 'file_name': file_name
```
Comment (aguschin) [inline reverse_image_search_module/search_image.py:62]: remove this?

File: reverse_image_search_module/search_image.py [hunk 3]
```diff
@@ -39,21 +58,47 @@ def change_format(data):
...
+    # plt.show()
+    #
+    #
+    # transposed_img = torch.from_numpy(np.transpose(img, (2, 0, 1)))
+    # #added
+    # transposed_img = transposed_img.to(torch.float)
+    # #
+    vector = img2vec.getVectors(img)
     vector = np.transpose(vector)
+    # vector = np.transpose(vector)
```
Comment (aguschin) [inline reverse_image_search_module/search_image.py:78]: remove?

File: reverse_image_search_module/search_image.py [hunk 4]
```diff
@@ -39,21 +58,47 @@ def change_format(data):
...
+
+# image_path = '../data/img/full/9247d9e1c6308bed661ba721f7f35be1fe1f3b52.jpg'
+# img = cv2.imread(image_path)
+# print(img.shape)
+# img = cv2.cvtColor(img, cv2.COLOR_BGR2RGB)
+# print(feature_vector.shape)
+#idx, dist, data = find_image(image)
+
+#print("Index:", idx)
+#print("Distance:", dist)
```
Comment (aguschin) [inline reverse_image_search_module/search_image.py:103]: create a test

Comment (aguschin) [reply reverse_image_search_module/search_image.py:103]: @michaelvin1322 did you wanted to create tests for this?

**Review by aguschin — CHANGES_REQUESTED** [review]

File: reverse_image_search_module/search_image.py (hunk 4 above)
Comment (mohammadsanaee) [reply reverse_image_search_module/search_image.py:103]: @mjason98 Alex told that you are writing the test
You can use this piece of code.

**Review by aguschin — APPROVED** [review]

**Pull request merged**